source/object-from-entries.ts

```diff
@@ -0,0 +1,36 @@
+/**
+A strongly-typed version of `Object.fromEntries()`.
+
+This is useful since `Object.fromEntries()` always returns `{ [k: string]: T }`. This function returns a strongly-typed object from given array of entries.
```

Comment:
```suggestion
This is useful since `Object.fromEntries()` always returns `{ [k: string]: T }`. This function returns a strongly-typed object from the given array of entries.
```